versions/next/asyncapi.md

```diff
@@ -17,24 +17,25 @@ The AsyncAPI Specification is a project used to describe and document Asynchrono
 The AsyncAPI Specification defines a set of files required to describe such an API.
 These files can then be used to create utilities, such as documentation, integration and/or testing tools.
 
-The file(s) MUST describe the operations a new [process](#definitionsProcess) can perform. For instance:
+The file(s) MUST describe the operations a new [application](#definitionsApplication) can perform. For instance:
 
 ```yaml
 user/signedup:
   subscribe:
     $ref: "#/components/messages/userSignUp"
 ```
 
-It means [processes](#definitionsProcess) can subscribe to `user/signedup` channel. However, it does NOT mean every [process](#definitionsProcess) must subscribe to this channel.
+It means [application](#definitionsApplication) can subscribe to `user/signedup` channel. However, it does NOT mean every [application](#definitionsApplication) must subscribe to this channel.
```

Comment:
So, following with the same reasoning, since an AsyncAPI document represents a single application, this sentence doesn't make sense anymore.

For context:
I wrote this sentence thinking about people who wanted to use AsyncAPI to document their whole event-driven microservices architecture. Not just a single service (application). But I realize it introduces more problems. That's why I suggested this feature: https://github.com/asyncapi/asyncapi/issues/146. So we make it clear the document represents the application.

Reply:
Agreed will clean up